app/src/main/java/src/main/java/io/apicurio/registry/rest/beans/ArtifactMetaData.java

```diff
@@ -0,0 +1,294 @@
+
+package io.apicurio.registry.rest.beans;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
Atm I've used jaxb, as Quarkus examples point in that direction.
But we can change it to Jackson ... if we know how. :-)

Reply:
I'll work on this as I evolve the jax-rs layer.  I've requested Jackson support for this release from the quarkus team and they've agreed.  I'm using a 3rd party library in Apicurio Studio to generate the bean classes from the JSON Schema data types defined in the OpenAPI spec.   We could switch to jaxb but I'd need to either user a diff. library or try to change the one I'm using to switch from jackson to jaxb...

Reply:
That said, our Data Model library in Apicurio (which we'll be using to process the OpenAPI and AsyncAPI artifacts) requires Jackson.  So we require Jackson in Quarkus for the registry anyway.

Reply:
Why requiring Jackson? Or why JAXB is not good enough?
Or Quarkus really doesn't support Jackson atm?